README.md

```diff
@@ -13,6 +13,9 @@ Dependencies:
 ```bash
 git clone https://github.com/vlang/py2v.git
 cd py2v
+
+# only use next command if you don't have V on PATH
+./v -prod json2v -o json2v/json2v
```

Comment:
```suggestion
# only needed if you don't have V on PATH
<path to V> -prod json2v -o json2v/json2v
```
People that don't have V on path will most likely won't have it in the py2v folder.